panels/info/cc-info-panel.c

```diff
@@ -1840,6 +1843,24 @@ cc_info_panel_init (CcInfoPanel *self)
       return;
     }
 
+  self->priv->hbox1 = WID ("hbox1");
+  self->priv->detail_vbox = WID ("detail_vbox");
+
+  if (gdk_screen_width () < 1024 || gdk_screen_height () < 600)
+    {
+      GtkWidget *sw;
+
+      sw = gtk_scrolled_window_new (NULL, NULL);
+      gtk_scrolled_window_set_min_content_height (GTK_SCROLLED_WINDOW (sw), 380);
+
+      gtk_container_remove (GTK_CONTAINER (self->priv->hbox1), self->priv->detail_vbox);
```

Comment:
Since we're retrieving the widget reference from the GtkBuilder instance which created this widget (the WID macro expands into a gtk_builder_get_object), the builder instance owns a reference widget.